src/utils/getLength.ts

```diff
@@ -8,10 +8,10 @@ import {pxToRem, fractionToPercent} from './units';
  * Other strings, e.g. those with a unit ('50%') will be passed through unchanged
  *
  * @param {number|string} inputLength - Value to be converted to a CSS length
- * @returns {string|number} - CSS length
+ * @returns {string|undefined} - CSS length
  */
 
-function getLength(inputLength) {
+function getLength(inputLength: number | string): string | number | undefined {
```

Comment:
I think your change to the JS Doc was right and this doesn't actually return a number. 🤔  Would have to look into it more closely to double-check, though.

```suggestion
function getLength(inputLength: number | string): string | undefined {
```

Reply:
ive removed it number return value and slightly changed the code.